### ISSUE TITLE
[Feature request]: 

### ISSUE THREAD

**Issue by lithium040** [LSS-Manager/LSSM-V.4#2906]
### Is your feature related to a problem?

No.

### Solution / Feature

It would be nice if there was another extension under 'General extension' to let the note be in a monospaced font (e.g. Courier New). It makes making tables way easier. The linkPreviews extension does this already on a new element, but does not change the original memo field.

### Additional

![image](https://github.com/LSS-Manager/LSSM-V.4/assets/35405557/ef968ee3-5f27-411b-8383-abfc08a07d9c)

**Comment by Cr4zyc4k3** [comment]
`font-family: monospace`
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

**Comment by lithium040** [comment]
Yes, and add that to the style element of the note-message container. I've looked into the DOM, but I'm totally oblivious with how LSSM is set up, with all that VUE and so on.

**Comment by Cr4zyc4k3** [comment]
We are currently discussing where this would fit best.
